src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -213,6 +226,10 @@ export const ArtworkSidebarExtraLinksFragmentContainer = createFragmentContainer
         }
         sale {
           is_closed: isClosed
+          is_benefit: isBenefit
```

Comment:
Why not just use `isBenefit` here? Doesn't seem like we gain much by aliasing. (I believe `is_closed: isClosed` was written that way for convenience during the migration 😄).

Reply:
Yeah I agree. Given that we have already migrated to MP v2 there isn't a lot of reasons we should continue to use lower case fields.